packages/components/RadioGroup/src/RadioButton.tsx

```diff
@@ -54,13 +61,16 @@ export const RadioButton = compose<IRadioButtonType>({
       if (!state.selected) {
         info.onChange && info.onChange(buttonKey);
       }
+      setIsFocused(true);
     }, [state, pressable.props, info, buttonKey]);
 
     const slotProps = mergeSettings<IRadioButtonSlotProps>(styleProps, {
       root: {
         rest,
+        ref: buttonRef,
         ...pressable.props,
         onFocus: onFocusChange,
+        onBlur: () => setIsFocused(false),
```

Comment:
You wouldn't need this either since pressable.props.onblur should be changing the focused state anyway

Reply:
Changed in newest commit